run-system-tests/action.yml

```diff
@@ -279,7 +291,8 @@ runs:
         (./zwallet version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZWallet Version Info:/") || true
         (./zbox version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZBox Version Info:/") || true
 
-    - name: "Set up wasm tests"
+    - name: "Set up frontend tests"
+      if:  ${{ env.RUN_FRONTEND_TESTS == 'true' }}
```

Comment:
it should be better to check if gosdk is changed

`if: ${{ inputs.custom_go_sdk_version }} != "" && ${{ inputs.custom_go_sdk_version }} != "NONE"`

Reply:
I disagree, this adds unnecessary complexity and could cause gaps in coverage, for example if gosdk was merged to staging with a bug
Easier and safer to always run for a given repo such as godsk imho where we can be sure they'll always run

Reply:
to compromise, I added both checks 
@cnlangzi 

if someone overrides gosdk on any repo, frontend tests will run